src/components/Button/CreateButton.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import {CreateButton} from './CreateButton.js';
+
```

Comment:
Tests always have similar code blocks. I have removed this issue from code climate.